Utilities/Config.cpp

```diff
@@ -65,6 +65,13 @@ bool cfg::try_to_int64(s64* out, const std::string& value, s64 min, s64 max)
 	const char* start = &value.front();
 	const char* end = &value.back() + 1;
 	int base = 10;
+	int sign = +1;
+
+	if (start[0] == '-')
```

Comment:
Need to make sign illegal if it appears again after 0x

Reply:
Added error check.